<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tir</class>
 <widget class="QWidget" name="Tir">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="posCursor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>posCursor</string>
   </property>
  </widget>
  <widget class="QLabel" name="WidgetZ">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>WidgetZ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label0">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>271</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hid detector</string>
   </property>
  </widget>
  <widget class="QLabel" name="hidDetector">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QLabel" name="hvalue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>hValue</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
